### PR TITLE
Translation [es]: Review

### PR DESCRIPTION
**Pull request by dieghernan**
I reviewed the translation fo Spanish. There were some typos and a mix of formal and informal expressions, that seemed odd to me.

I also tried to make the translation more inclusive (in Spanish most adjectives and roles are different depending of the gender).

It would be great if someone from Latin America can have a look as well to check that is not too awkward for them (I tried to do my best but obviously I am biased towards Spanish from Spain).

Regards

### CODE REVIEW

**Comment by rivaquiroga** [comment]
Hi, Diego! I'm a native Spanish speaker from Latin America. I'll take a look :)

File: po/R-es.po [hunk 1]
```diff
@@ -14,83 +14,83 @@ msgstr ""
...
 
 #: build-home-authors.R:203
 msgid "contributor"
-msgstr "colaborador"
+msgstr "colaborador(a)"
 
 #: build-home-authors.R:204
 msgid "copyright holder"
-msgstr "dueño de los derechos de autor"
+msgstr "propietario de los derechos de autor"
```
Comment (rivaquiroga) [inline po/R-es.po:65]: quizás acá habría que hacer la duplicación de género gramatical también, para mantener la consistencia con los otros roles

File: po/R-es.po [hunk 2]
```diff
@@ -14,83 +14,83 @@ msgstr ""
...
 
 #: build-home-authors.R:206
 msgid "data contributor"
-msgstr "datos proporcionados por"
+msgstr "colaborador(a) de datos"
 
 #: build-home-authors.R:207
 msgid "funder"
-msgstr "fundador"
+msgstr "patrocinador"
```
Comment (rivaquiroga) [inline po/R-es.po:77]: lo mismo acá. También duplicaría el género

File: po/R-es.po [hunk 3]
```diff
@@ -168,27 +168,27 @@ msgstr "Tutoriales"
...
+msgstr "Versión lanzada"
 
 #: development.R:21
 msgid "In-development version"
 msgstr "Versión en desarrollo"
 
 #: development.R:22
 msgid "Unreleased version"
-msgstr "Versión no en CRAN"
+msgstr "Versión en desarrollo"
```
Comment (rivaquiroga) [inline po/R-es.po:179]: Esta traducción quedó justo igual a la anterior. Quizás podría ser "Versión no lanzada" o "Versión no publicada"

File: po/R-es.po [hunk 4]
```diff
@@ -168,27 +168,27 @@ msgstr "Tutoriales"
...
 
 #: navbar.R:97
 msgid "Reference"
 msgstr "Ayuda"
```
Comment (rivaquiroga) [inline po/R-es.po:184]: Si esta sería la traducción para la lista de funciones, no sé si con "ayuda" quede tan claro. Podría ser "Referencia"

File: po/R-es.po [hunk 5]
```diff
@@ -221,7 +221,7 @@ msgstr "Autor"
```
Comment (rivaquiroga) [inline po/R-es.po:221]: Para tener consistencia, también habría que duplicar el género "Autor"

File: po/R-es.po (hunk 3 above)
Comment (hadley) [reply po/R-es.po:179]: This message is relatively rarely used, but it is meant to indicate that it's a development version of a package that has never been released to CRAN (as opposed to development version that's newer than the version on CRAN).

File: po/R-es.po [hunk 6]
```diff
@@ -168,27 +168,27 @@ msgstr "Tutoriales"
 #: development.R:20
 #, fuzzy
 msgid "Released version"
-msgstr "Version official"
+msgstr "Versión lanzada"
```
Comment (rivaquiroga) [inline po/R-es.po:171]: ¿Qué opinas de "Versión publicada"? No es exactamente lo mismo, pero "Versión lanzada" no es una expresión muy frecuente.

File: po/R-es.po (hunk 4 above)
Comment (hadley) [reply po/R-es.po:184]: For more context, this is the link in the navbar that takes you to the index that lists all `.Rd` files. I don't think "reference" is particularly clear in English, either, but it's the best we could come up with.

File: po/R-es.po (hunk 6 above)
Comment (hadley) [reply po/R-es.po:171]: "Published version" makes sense to me, and then "unpublished version" is more clear than "Unreleased version".

File: po/R-es.po (hunk 3 above)
Comment (rivaquiroga) [reply po/R-es.po:179]: My issue here is that the same translation is proposed for "In-development version" and "Unreleased version"

File: po/R-es.po (hunk 4 above)
Comment (rivaquiroga) [reply po/R-es.po:184]: My concern about "Ayuda" is that people may think they will find help there. @dieghernan, what do you think of something like "guía", "guía de referencia", "índice"?

File: po/R-es.po (hunk 6 above)
Comment (rivaquiroga) [reply po/R-es.po:171]: Agree!

File: po/R-es.po (hunk 3 above)
Comment (dieghernan) [reply po/R-es.po:179]: Oh! I didn’t realise, bien visto @rivaquiroga

File: po/R-es.po (hunk 4 above)
Comment (dieghernan) [reply po/R-es.po:184]: I like índice, as in a book

File: po/R-es.po (hunk 6 above)
Comment (dieghernan) [reply po/R-es.po:171]: Much better, like it

**Comment by dieghernan** [comment]
For info, I put this on hold as I detected an issue on translation. I would send a PR that should be merged before this one

**Comment by dieghernan** [comment]
On hold waiting for #1906

**Comment by dieghernan** [comment]
On hold, see #1906

File: po/R-es.po (hunk 1 above)
Comment (dieghernan) [reply po/R-es.po]: I replaced this with "titular de los derechos de autor"

En España este es el término legal, pero me parecía apropiado ya que estamos hablando de derechos de autor. ¿Que te parece @rivaquiroga ?

File: po/R-es.po (hunk 2 above)
Comment (dieghernan) [reply po/R-es.po]: Hecho!

File: po/R-es.po (hunk 3 above)
Comment (dieghernan) [reply po/R-es.po]: "Versión no publicada". Al final:

```

#: development.R:20
msgid "Released version"
msgstr "Versión publicada"

#: development.R:21
msgid "In-development version"
msgstr "Versión en desarrollo"

#: development.R:22
msgid "Unreleased version"
msgstr "Versión no publicada"

```

File: po/R-es.po (hunk 5 above)
Comment (dieghernan) [reply po/R-es.po:229]: Hecho!

File: po/R-es.po (hunk 6 above)
Comment (dieghernan) [reply po/R-es.po]:
```

#: development.R:20
msgid "Released version"
msgstr "Versión publicada"

#: development.R:21
msgid "In-development version"
msgstr "Versión en desarrollo"

#: development.R:22
msgid "Unreleased version"
msgstr "Versión no publicada"


```

File: po/R-es.po (hunk 4 above)
Comment (dieghernan) [reply po/R-es.po:193]: Finalmente: Índice

File: po/R-es.po (hunk 1 above)
Comment (rivaquiroga) [reply po/R-es.po]: ¡Mucho mejor!

**Comment by hadley** [comment]
Thanks for all your work on this!

**Pull request merged**